apps/server/src/modules/authorization/types/action.enum.ts

```diff
@@ -1,3 +1,4 @@
+// need to move to shared
```

Comment:
Können wir die Regel, dass TODO-Kommentare beanstandet werden, in Sonarcloud nicht einfach ausstellen? Das hier ist doch ein klarer TODO-Kommentar, wo einfach nur das TODO weggelassen ist. Das macht sie aber schwerer zu finden und schwerer zu unterscheiden,

Reply:
Ist bereits entfernt.